examples/run_examples.sh

```diff
@@ -19,41 +19,35 @@
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
-set -e
-
-SKIP_LIST="
-    MyModule/sli/example.sli
-    Potjans_2014/spike_analysis.py
-    Potjans_2014/user_params.sli
-    ReadData_demo.sli
-    music/
-    nestrc.sli
-    neuronview.py
-    plot_tsodyks_depr_fac.py
-    plot_tsodyks_shortterm_bursts.py
-"
-
-FAILURES=0
+NEST_CMD=`which nest`
+if [ $? != 0 ] ; then
+    echo "ERROR: command 'nest' not found. Please make sure PATH is set correctly"
+    echo "       by sourcing the script nest_vars.sh from your NEST installation."
+    exit 1
+fi
 
-# Trim leading and trailing whitespace and make gaps exactly one space large
-SKIP_LIST=$(echo $SKIP_LIST | sed -e 's/ +/ /g' -e 's/^ *//' -e 's/ *$//')
+python -c "import nest" >/dev/null 2>&1
+if [ $? != 0 ] ; then
+    echo "ERROR: PyNEST is not available. Please make sure PYTHONPATH is set correctly"
+    echo "       by sourcing the script nest_vars.sh from your NEST installation."
+    exit 1
+fi
 
-# Create a regular expression for grep that removes the excluded files
-case "$SKIP_LIST" in  
-    *\ * ) # We have spaces in the list
-        SKIP='('$(echo $SKIP_LIST | tr ' ' '|' )')' ;;
-    *)
-        SKIP=$SKIP_LIST ;;
-esac
+FAILURES=0
 
-# Find all examples in the installation directory
-EXAMPLES=$(find ${SEARCH_DIR:-./examples} -type f -name \*.py -o -name \*.sli | sort -t. -k3)
+# Find all examples in the installation directory that have a line
+# containing "autorun=true"
+EXAMPLES=$(grep -rl --include=\*\.sli 'autorun=true' nest/)
```

Comment:
This only works in source code. In an installation `run_examples.sh` is in  `NEST_INSTALL_DIR/share/doc/nest/`. There the examples are in `examples/` not `nest/`. See #1138.

Reply:
@steffengraber I have addressed this, please re-review.